clients/cpp/samples/ClientSample/ClientSample.vcxproj

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Look at files in folder here: https://github.com/SignalR/SignalR-Client-Cpp/tree/dev/Build - they help avoid duplication. I don't want to go too far but I think you will want to copy a lot of this. Not only because they worked but they also use compiler settings required/recommended by security folks.